<launch>
				<!--  ************** Odometry ***************  -->
	<!-- <param name="robot_description" command="cat $(find nox_description)/urdf/nox.urdf" />
	
	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
	
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" /> -->
    <!-- <node pkg="tf" type="static_transform_publisher" name="base_f_to_l" 
    args="0.0 0.0 0.0 0.0 0.0 0.0 /base_footprint /base_link 40" /> -->
       <node pkg="tf" type="static_transform_publisher" name="base_f_to_laser" 
    args="0.09 0.0 0.28 0.0 0.0 0.0 /base_link /laser 30" />
	<node name="serial_node" pkg="rosserial_python" type="serial_node.py">
		<param name="port" value="/dev/mega2560"/>
	</node>
	<!-- <node name="mpu_node" pkg="nox" type="mpu6050.py" output="screen">
		<param name="odom_topic" value="odom"/>
	</node> -->
	<node name="nox_controller" pkg="nox" type="nox_controller">
	<param name="initpose_file_path" value="/home/ubuntu/amr_ws/src/nox/depends/init_pose.json" />
		<param name="publish_tf" value="true" />
		<param name="robot_width" value="0.314" />
    		<param name="publish_rate" value="20.0" />
    		<param name="GEAR_RATIO" value="2.065" />
			<param name="print_data" value="false" />
			<param name="print_yaw" value="true" />
			<param name="use_mpu" value="false" />
    		<param name="bias" value="0.0" />
    		<param name="p_rot" value="0.295" />
    		<param name="d_rot" value="0.2" />
	</node>	 
				<!--  ************** Sensors ***************  -->
	 <include file="/home/ubuntu/amr_ws/src/ydlidar_ros_driver/launch/X2.launch"/>
	<!-- node name="rviz" pkg="rviz" type="rviz" args="-d $(find nox_description)/urdf.rviz" required="true" / -->
</launch>
